databrowser/src/components/languageSwitcher/LanguageFilter.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <button
+    class="
+      inline-flex
+      md:hidden
+      items-center
+      py-1
+      px-4
+      rounded-full
+      border border-gray-500
+    "
+    @click="showMobileSelect = true"
+  >
+    <span class="pr-2">{{ selected }}</span>
+    <ArrowDown />
+  </button>
+  <div class="hidden md:inline-flex">
+    <div
+      v-for="(language, index) in languages"
+      :key="language"
+      :class="{
+        'text-green-500 bg-opacity-10 bg-green-500 border-green-500':
+          isSelected(language),
+        'hover:bg-gray-300': !isSelected(language),
+        'rounded-l-full border-l': index == 0,
+        'rounded-r-full border-r': index == languages.length - 1,
+      }"
+      class="
+        overflow-hidden
+        last:pr-1
+        first:pl-1
+        border-t border-b border-gray-500
+      "
+    >
+      <div
+        :class="{
+          'border-green-500': isSelected(language),
+          'border-l': index != 0,
+          'border-r': index != languages.length - 1,
+        }"
+        class="border-transparent"
+        role="button"
+        tabindex="0"
+        @click="changeLanguage(language)"
+      >
+        <span class="block py-1 px-2 font-semibold">{{ language }}</span>
+      </div>
+    </div>
+  </div>
```

Comment:
This should be a shareable component. The pagination has the exact same style.

Maybe you can call it `ButtonPillGroup`?

https://getbootstrap.com/docs/4.0/components/button-group/